lib/taskana-core/src/main/java/pro/taskana/workbasket/internal/WorkbasketServiceImpl.java

```diff
@@ -829,6 +849,29 @@ public void deleteWorkbasketAccessItemsForAccessId(String accessId)
     }
   }
 
+  /**
+   * Check if current object is based on the newest (by modified).
+   *
+   * @param workbasketImplToUpdate the workbasket to update
+   * @throws ConcurrencyException if the workbasket has been modified by some other process.
+   * @throws WorkbasketNotFoundException if the given workbasket does not exist.
+   */
+  private void checkIfWorkbasketExistsAndVerifyTimestampHasNotChanged(
```

Comment:
please refactor this method to get both Workbaskets as parameters. 
this will make it testable and independent from the database.

Reply:
Done